### ISSUE TITLE
Error compiling rustless

### ISSUE THREAD

**Issue by dariusc93** [rustless/rustless#53]
I've been testing out rustless for a while but the latest version currently does not compile due to the following error:

```
native library `openssl` is being linked to by more than one package, and can only be linked to by one package

  openssl-sys v0.6.7
  openssl-sys v0.7.0

```

**Comment by s-panferov** [comment]
@dariusc93 this is because https://github.com/iron/iron/commit/32f8dfc65c87949c5a6b55a168298c81661cce1d. Everything is fixed in v0.7.2

**Comment by s-panferov** [comment]
Thanks for your report!
